tests/OpenSSLErrorTest.cpp

```diff
@@ -507,10 +575,6 @@ TEST(OpenSSLErrorTest, RS256VerifyErrorCode) {
     });
 }
 
-TEST(OpenSSLErrorTest, ECDSAKeyReference) {
```

Comment:
Is there a reason why you remove the reference test for ECDSAKey ?

Reply:
This is already done the token tests and it's not  an "openssl error" test, plus it's the only key reference test